README.md

```diff
@@ -6,7 +6,7 @@ A Binder-compatible repo with an `environment.yml` file.
 
 Access this Binder by clicking the blue badge above or at the following URL:
 
-http://mybinder.org/v2/gh/binder-examples/conda_environment/master?filepath=index.ipynb
+http://mybinder.org/v2/gh/binder-examples/conda_environment/main?filepath=index.ipynb
```

Comment:
```suggestion
http://mybinder.org/v2/gh/binder-examples/conda_environment/HEAD?filepath=index.ipynb
```
In the (unlikely) event the default branch changes again `HEAD` will remain correct.

Please could you also modify the link in the `[Binder]` badge at the top?